.github/workflows/integration_test.yml

```diff
@@ -32,7 +32,7 @@ jobs:
 
       - name: Task pre integration test
         env:
-          STRING_TO_CHANGE: '\(pallet-parachain-staking.*\)\(".*\)"\(.*branch.*\)\(".*"\)\(.*\)'
+          STRING_TO_CHANGE: '\(.*\)\("https://github.com/Aventus-Network-Services/avn-parachain"\)\(.*\)\(branch.*\)\(".*"\)\(.*\)'
```

Comment:
just to be 100% safe, should we add the `git = ` prefix on the url? Something like `git\s*=\s*` maybe?

Reply:
discussed offline and decided to go without these changes ☝️